.github/workflows/changelog_generation.yml

```diff
@@ -20,13 +20,15 @@ jobs:
           go-version: 1.17
         id: go
 
-      - run: make changelog
+      - run: make changelog vendor
       - name: Commit & Push changes
         run: |
           if [[ ! -z  $(git status --porcelain) ]]; then
             git config --local user.email elasticcloudclients@elastic.co
             git config --local user.name elasticcloudclients
             git add CHANGELOG.md
             git commit -m "Update CHANGELOG.md"
+            git add NOTICE
+            git commit -m "Update NOTICE"
```

Comment:
I'm not sure this approach will work very well when there aren't any changelog updates but we need to re-generate the NOTICE file. It's probably not a big deal, but this means that the NOTICE file will only be ever updated when the changelog has new entries.

```console
$ git commit -m "test"; echo $?
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean
1
```

I would instead do add both files on the `git add` on line 29 and commit both files with an updated commit message.

Reply:
Fixed